### PR TITLE
[RJA-1514][feature] - Remove deprecated api.RequestResult

### PR DESCRIPTION
**Pull request by rsvato**

### CODE REVIEW

File: backend/src/main/scala/com/griddynamics/genesis/configuration/TemplateServiceContext.scala [hunk 1]
```diff
@@ -47,7 +47,7 @@ class GroovyTemplateServiceContext {
     @Bean(name = Array("groovy")) def templateService = new GroovyTemplateService(
         templateRepositoryContext.templateRepository,
         stepBuilderFactories ++ genericStepDefinitions.map(definition => new ReflectionBasedStepBuilderFactory(definition.name, definition.step)),
-        ConversionServiceFactory.createDefaultConversionService(),
+        new DefaultConversionService,
```
Comment (ddurnev) [inline backend/src/main/scala/com/griddynamics/genesis/configuration/TemplateServiceContext.scala:50]: Why not use ConversionServiceFactory?

Comment (rsvato) [inline backend/src/main/scala/com/griddynamics/genesis/configuration/TemplateServiceContext.scala:50]: deprecated

Comment (rsvato) [inline backend/src/main/scala/com/griddynamics/genesis/configuration/TemplateServiceContext.scala:50]: http://static.springsource.org/spring/docs/3.1.x/javadoc-api/org/springframework/core/convert/support/ConversionServiceFactory.html#createDefaultConversionService()

Comment (ddurnev) [inline backend/src/main/scala/com/griddynamics/genesis/configuration/TemplateServiceContext.scala:50]: import of ConversionServiceFactory is useless?

Comment (rsvato) [inline backend/src/main/scala/com/griddynamics/genesis/configuration/TemplateServiceContext.scala:50]: updated

Comment (ddurnev) [inline backend/src/main/scala/com/griddynamics/genesis/configuration/TemplateServiceContext.scala:50]: In TemplateServiceContext?

Comment (rsvato) [inline backend/src/main/scala/com/griddynamics/genesis/configuration/TemplateServiceContext.scala:50]: Probably we have to create an issue on optimizing imports across all project and assign it to you

**Pull request closed without merging**